features/trig-functions.yml.dist

```diff
@@ -2,8 +2,9 @@
 # Do not edit this file by hand. Edit the source file instead!
 
 status:
-  baseline: low
+  baseline: high
   baseline_low_date: 2023-03-13
+  baseline_high_date: 2025-09-13
```

Comment:
Also hard to search, but I didn't see any bugs on BCD or the vendor trackers that suggested that BCD's wrong here. Allowing this to advance.